packages/event-handler/src/rest/constants.ts

```diff
@@ -10,6 +10,18 @@ export const HttpVerbs = {
   OPTIONS: 'OPTIONS',
 } as const;
 
+export const HttpErrorCodes = {
```

Comment:
A consideration I had here was if we were add every possible HTTP error code to this object, we could make incorrect HTTP codes unrespresentable via the the type system. Thoughts?

Reply:
Do you mean adding specific entries to this type for things like `400`, `401`, `403`, etc.? Is it worth it?

Reply:
I see, well since you already have the list I'd say let's add it.